Assets/LeapMotion/Scripts/HandPool.cs

```diff
@@ -15,37 +16,61 @@ namespace Leap.Unity {
    */
   public class HandPool :
     HandFactory {
-
+    public bool EnforceHandedness = false;
     [SerializeField]
     private List<ModelPair> ModelCollection;
     [SerializeField]
-    private List<ModelGroup> ModelPool;
-    public bool EnforceHandedness = false;
-    
+    public List<ModelGroup> ModelPool;
+    public List<HandRepresentation> ActiveHandReps = new List<HandRepresentation>();
```

Comment:
These variables should not be publicly modifiable, since any changes could disrupt the entire pooling mechanism.